suggest `*.desktop` for AppImage (#10823)
helix-editor/helix: book/src/package-managers.md

chmod +x helix-*.AppImage # change permission for executable mode
./helix-*.AppImage # run helix
```

> You can optionally [add the `.desktop` file](./building-from-source.md#configure-the-desktop-shortcut), but there's some pre-requisites:

Reviewer: ```suggestion
You can optionally [add the `.desktop` file](./building-from-source.md#configure-the-desktop-shortcut). Helix must be installed in `PATH` with the name `hx`. For example:
```

```sh

> mkdir -p "$HOME/.local/bin" && \
> mv helix-*.AppImage "$HOME/.local/bin/hx"

Reviewer: ```suggestion
mkdir -p "$HOME/.local/bin"
mv helix-*.AppImage "$HOME/.local/bin/hx"
```

I think it's clearer if these are separate commands

Author: I guess? It looks cleaner without `&&`. I added `&&` for safety and performance, but I guess that's irrelevant here, especially considering `mv` should fail anyways if the dir is non-existent

Reviewer: yeah that was my thinking as well, if `mkdir` fails then `mv` certainly will too

```

> `hx` can be anywhere in your `$PATH`, but (by [convention](https://askubuntu.com/a/1408444)) `~/.local/bin` is preferred.

Reviewer: ```suggestion
and make sure `~/.local/bin` is in your `PATH`.
```

justifying the use of `~/.local/bin` seems unnecessary/offtopic to me, especially if we frame the directions above as an example
